src/main/ui/app/component/AddOnSearch.jsx

```diff
@@ -2,7 +2,7 @@ import {Component} from "react";
 import fetch from "isomorphic-fetch";
 import AddOnList from "./AddOnList";
 import DebounceInput from "react-debounce-input";
-import { DropdownButton , MenuItem } from 'react-bootstrap';
+import { DropdownButton , MenuItem , Glyphicon} from 'react-bootstrap';
```

Comment:
I was going to add a search icon to it but then it didn't look all that good so I removed it .The gist is that this is no longer needed and I'll revert that change ;)